src/libmscore/fret.cpp

```diff
@@ -1333,6 +1333,7 @@ void FretDiagram::writeMusicXML(XmlWriter& xml) const
             xml.tag("fret", "0");
             xml.etag();
         } else {
```

Comment:
Careful, I explicitly removed this else statement for a reason in original PR :)

Reply:
oops

Reply:
Better now?